src/systems/userinput/bindings/keyboard-mouse-user.js

```diff
@@ -116,6 +138,26 @@ export const keyboardMouseUserBindings = {
         value: paths.actions.logDebugFrame
       },
       xform: xforms.rising
+    },
+    {
+      src: {
+        value: paths.device.mouse.buttonRight
+      },
+      dest: {
+        value: paths.actions.startGazeTeleport
```

Comment:
It ought to work the way you have in mind, but that is not currently supported because `teleport-controls` wants to shoot the ray out from an object3D, and the mouse-and-keyboard user doesn't have an object3D to put a `teleport-controls` onto that would behave the way we would need it to. This was a fast way to enable teleporting without revisiting `teleport-controls`. (If instead I just bind this button to teleport with the right hand, the right hand controller object is pointed straight downward and I teleport to the floor below me.)